Easydict/App/InfoPlist.xcstrings

```diff
@@ -0,0 +1,60 @@
+{
+  "sourceLanguage" : "en",
+  "strings" : {
+    "CFBundleDisplayName" : {
+      "comment" : "Bundle display name",
+      "extractionState" : "extracted_with_value",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "new",
+            "value" : "Easydict"
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Easydict"
+          }
+        }
+      }
+    },
+    "CFBundleName" : {
+      "comment" : "Bundle name",
+      "extractionState" : "extracted_with_value",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "new",
+            "value" : "Easydict"
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Easydict"
+          }
+        }
+      }
+    },
+    "NSHumanReadableCopyright" : {
+      "comment" : "Copyright (human-readable)",
+      "extractionState" : "extracted_with_value",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "new",
```

Comment:
Shall we change it to `translated`?

Reply:
I'm not sure why it's `new` instead of `translated` here, but Xcode doesn't warn about it, so it doesn't seem to be a problem.

`InfoPlist.xcstrings` looks a bit different from `Localizable.xcstrings`, I notice that all en states are `new`, zh-Hans state is `translated`.

<img width="1412" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/7c19b267-40c7-47b5-aa38-a92a6a1ae822">

Reply:
I recreated the `InfoPlis`t and tested this situation, and found that the extractionState of `InfoPlist.xcstrings` is `extracted_with_value` by default, while the state is `new`.

 If the value of extractionState is `manual`, the state is `translated`, which is the case for `Localizable.xcstrings`.

<img width="906" alt="image" src="https://github.com/tisfeng/Easydict/assets/25194972/ec5b6738-c573-42e9-a6e4-910d8f240965">

Reply:
Do we need to change `extractionState` to `manual` or use the default value

Reply:
It sounds like `.xcstrings` may have the same behavior, but do we need to change `extractionState` to `manual`, Xcode isn't warning about it.

Reply:
It doesn't seem to have any impact, this may be a minor issue with Xcode.

If we want to change `extractionState` to `manual`, we need to open the file in source code mode, which is a bit tedious. I think we can leave it as it is.

Reply:
Yeah, I think changing it may be a bit complicated